service/cloudFormation/sam-template-empty.yml

```diff
@@ -18,7 +18,7 @@ Resources:
       Description: Lambda Function to run playwright test
```

Comment:
What is the difference between this file (sam-template-empty), and the other template? Where do we use each one of them?